package.json

```diff
@@ -135,6 +136,7 @@
 		"postpack": "rm -f oclif.manifest.json",
 		"lint": "eslint ./src --ext .ts",
 		"test": "./gradlew test jacocoTestCoverageVerification && nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
+		"test-quiet": "cross-env SFGE_LOGGING=false ./gradlew test jacocoTestCoverageVerification && nyc mocha --timeout 10000 --retries 5 \"./test/**/*.test.ts\"",
```

Comment:
Using the new variable, we can disable SFGE's internal logging, which is important to avoid overflowing the logs in Github Action.

Reply:
I can see why this is important. Though when we have failures, the logs would surely help with troubleshooting.
How about writing the output to a file instead of letting it overflow to the console window?

Reply:
My assumption is that, for a test failure, you'd check the artifact to see what the failures were, and then run those specific tests locally to reproduce.

Reply:
I suppose my discomfort comes from situations where the issue doesn't repro locally. Can we set a higher threshold for logging rather than completely turning it off?

Reply:
I'll see how doable that is and get back to you.